cypress/integration/userCanCreateTimeRecording.feature.js

```diff
@@ -4,43 +4,50 @@ describe("User can create time recording", () => {
 	it("successfully", () => {
 		cy.server();
 		cy.route({
-			method: 'POST',
-			url: 'https://demo-stable.kimai.org/api/timesheets',
-			response: 'fixtures:save_data.json',
+			method: "POST",
+			url: "https://demo-stable.kimai.org/api/timesheets",
+			response: "fixtures:save_data.json",
 			headers: {
 				"X-AUTH-USER": "susan_super",
 				"X-AUTH-TOKEN": "api_kitten"
 			}
-		})
-		cy.visit("http://localhost:3000")
-		cy.get('input[type=name]').type('anna_admin')
-		cy.get('input[type=password]').type('api_kitten')
-		cy.get('button[type=submit]').click()
-		cy.get('input[name="begin"]').type("07:00")
-		cy.get('input[name="end"]').type("08:00")
-		cy.get('button[name="create"]').click()
-		cy.contains('Your time was saved')
-	})
+		});
+		cy.visit("http://localhost:3000");
+		cy.get("input[type=name]").type("anna_admin");
+		cy.get("input[type=password]").type("api_kitten");
+		cy.get("button[type=submit]").click();
+		cy.get("#customer > .dropdown").click();
+		cy.contains("Hammes-Kilback").click();
+		cy.get("#projects > .dropdown").click();
+		cy.contains("Advanced content-based functionalities").click();
+		cy.get('input[name="begin"]').type("07:00");
+		cy.get('input[name="end"]').type("08:00");
+		cy.get(':nth-child(9) > input').type("25");
```

Comment:
What is this field? I can't make it out...

Reply:
it's from that target selector in the cypress program, where u can select a specific field and u get a "sample" basically, It's the Rate input field.

Reply:
I understand that. There must be a more readable way to identify that selector

Reply:
Fixed.